Lexis+.js

```diff
@@ -40,7 +40,7 @@ function detectWeb(doc, _url) {
 		return "multiple";
 	}
 	else if (/[a-zA-Z. ]+\s§\s\d+/.test(doc.title)
-	|| /\W*(act)\W|\W*(act)$/i.test(doc.title)
+	|| /\W(act)\W|\W(act)$|\W(acts)\W|\W(acts)$/i.test(doc.title)
```

Comment:
How about just `/\W(acts?)(\W|$)/` ?

Also good to add a comment with some real-life examples.

Reply:
Perfect, thanks!